julia/Related/src/Related.jl

```diff
@@ -98,6 +97,8 @@ end
 function main()
     json_string = read(@__DIR__()*"/../../../posts.json", String)
     posts = JSON3.read(json_string, Vector{PostData})
+    fake_posts = fill(posts[1], length(posts))
```

Comment:
for reviewer: this just repeat the `posts[1]` N times where N is the real length.